### DISCUSSION TITLE
Help integrating with Microsoft Teams

### DISCUSSION THREAD

**Discussion by mrmoosefish** [DependencyTrack/dependency-track#1666] [Q&A] [answered]
Is there more documentation available for integrating notifications with Microsoft Teams? Or more simply, does it require admin access to Teams to configure?

**Comment by stevespringett** [comment, marked as answer]
It's been a few years since support for teams were added. But I'm pretty sure I followed the instructions Microsoft provides.

https://docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook
